_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -6467,8 +6467,8 @@
 /obj/machinery/door/firedoor,
 /obj/structure/window/reinforced,
 /obj/machinery/computer/cargo/request{
-	icon_state = "computer";
-	dir = 1
+	dir = 1;
+	icon_state = "computer"
```

Comment:
Fucking monster

Reply:
this is why you use sdmmlauncher

Reply:
Why the fuck is meta even being touched. Never touched it. Goddammit. Gimme a second.